src/nanoemoji/write_test_glyphmap.py

```diff
@@ -0,0 +1,53 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Test glyphmap writer. Writes a glyphmap where every second glyph gets no codepoints.
```

Comment:
i don't think this should be located inside nanoemoji namespace, it'd be unnecessarily installed alongside the rest of the lib and it's basically only useful for self testing. It should be located in the tests/ directory

Reply:
Hm, yes, I think you are right. Setting PYTHONPATH (I assume I can set env vars for the spawned process) will probably work.

Reply:
Fixed; we now rig the environment for the subprocess